dae-alright/src/components/Navigation/Navigation.scss

```diff
@@ -0,0 +1,33 @@
+.logo{
+    margin-left: 5%;
```

Comment:
Ah, why not give the `nav` tag an extra class that gives it a `flex property `and `space things around` with `justify-content: space-around`? This should do the trick rather than explicitly setting `margin-left` and `margin-right` on the `logo` and `shopping-cart` classes respectively.

Reply:
OH Ok.